anitya/auth.py

```diff
@@ -0,0 +1,59 @@
+import flask
```

Comment:
This file deserves some comments. :-)

Reply:
I saw that the social_auth stuff was registered as flask blueprint in the `app.py`, so I was trying to make it similar.
The rest is from authlib examples. The `def oauthlogin` is usually named `login` in the authlib documentation. I renamed it because there already is login function in ui.py.
Then I created the tokens in github and google for the anitya app and since both parties send a different user_info, so it is processed differently. 
I will add docstrings and comments.